### ISSUE TITLE
React.createContext on server

### ISSUE THREAD

**Issue by revskill10** [fusionjs/fusion-apollo#111]
### Type of issue

Feature request and bug

### Description

Due to this [PR](https://github.com/apollographql/react-apollo/pull/2304), we need to keep track of when using `ReactContext` universally.

**Comment by KevinGrandon** [comment]
I think we already support using context for SSR, and the react-apollo PR is just fixing some of their usage of it? Anything specific we need to do here?

**Comment by revskill10** [comment]
@KevinGrandon Currently , the context itself doesn't work on SSR, it needs to be fixed on `react-apollo`, we can't do anything, sorry.

**Comment by revskill10** [comment]
Our CacheContext will work fine as long as it's on client.

**Comment by KevinGrandon** [comment]
Thanks, I'm still not clear on what we need to do in our codebase. Do we just need to upgrade react-apollo versions when it comes out?

**Comment by KevinGrandon** [comment]
I believe we just need to upgrade react-apollo once the fix is ready, so closing this for now. Feel free to re-open or file a new issue if there's anything specific we can do in the meantime. Thanks for bringing this to our attention.

**Comment by revskill10** [comment]
@KevinGrandon This issue has been resolved, cheer
https://github.com/apollographql/react-apollo/issues/2291
